lib/reports/etas_organization_overlap_report.rb

```diff
@@ -18,10 +18,10 @@ def initialize(organization = nil)
       @local_report_path = Settings.local_report_path || "local_reports"
       Dir.mkdir(@local_report_path) unless File.exist?(@local_report_path)
       # persistent storage
-      @persistent_report_path = Settings.etas_overlap_reports_path || "tmp_reports"
+      @persistent_report_path = Settings.etas_overlap_reports_path
```

Comment:
Should use the settings mechanism rather than setting it in a surprising way here. Should probably update `@local_report_path` above as well.

Reply:
Will finish cleaning this up in DEV-406 / DEV-407